.circleci/config.yml

```diff
@@ -200,7 +204,7 @@ jobs:
                 role-arn: $AWS_ROLE_ARN
                 role-session-name: "CircleCI-${CIRCLE_WORKFLOW_ID}-${CIRCLE_JOB}"
             - get_workspace
-            - install_container_dependencies
+            - install_npm
```

Comment:
npm is already installed in the image. You shouldn't need to do this, unless you need a specific version of npm.